stdlib/pervasives.gr

```diff
@@ -97,6 +99,11 @@ let (isnt) : (a, a) -> Bool = (x, y) => !(x is y)
 
 export enum List<a> { [], [...](a, List<a>) }
 
+/**
+ * @deprecated This will be removed in a future release of Grain.
+ */
+let cons = (a, b) => [a, ...b] // <- workaround for (grain-lang/grain#802) [TODO] fix #802 and delete
```

Comment:
This sorta makes me want to add `empty` as well. Feels awkward just to have `cons`.